content/docs/lists-and-keys.md

```diff
@@ -202,13 +202,12 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
-
-A good rule of thumb is that elements inside the `map()` call need keys.
+[**इसे CodePen पर आज़माएँ**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
```

Comment:
A good rule of thumb is that elements inside the `map()` call need keys. -> statement has been removed altogether?